specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/examples/ForceDeleteVirtualMachine.json

```diff
@@ -3,7 +3,7 @@
     "subscriptionId": "{subscription-id}",
     "resourceGroupName": "myResourceGroup",
     "vmName": "myVM",
-    "forceDeletion": "true",
+    "forceDeletion": true,
```

Comment:
Is this an intended change?

Reply:
yes Kashif, it is intended change. The forcedeletion datatype marked as boolean but specified as string which was not caught by dev or reviewers in the PR where this example was added. when i ran the validate-example command, this was found. so fixed it.